MachOKit/ObjC/MKObjCElementList.m

```diff
@@ -155,7 +155,7 @@ + (Class)classForGenericArgumentAtIndex:(__unused NSUInteger)index
 
 //|++++++++++++++++++++++++++++++++++++|//
 + (uint32_t)flagsMask
-{ return 0; }
+{ return 3; }
```

Comment:
What is the reason for this change?  According to the ObjC runtime source, only `method_list_t` reserves the least significant two bits for flags.  `MKObjCClassMethodList` should override `+flagsMask` to return 3 instead of changing the default implementation.

https://opensource.apple.com/source/objc4/objc4-781/runtime/objc-runtime-new.h.auto.html

Reply:
My bad, I've corrected this in fd20729.